inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/TruncateFunction.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sdk.transform.process.function;
+
+import org.apache.inlong.sdk.transform.decode.SourceData;
+import org.apache.inlong.sdk.transform.process.Context;
+import org.apache.inlong.sdk.transform.process.operator.OperatorTools;
+import org.apache.inlong.sdk.transform.process.parser.ValueParser;
+
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.Function;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+/**
+ * TruncateFunction
+ * description: returns the number that intercepts integer2 decimal places.
+ *              If numeric1 or integer2 is NULL, NULL is returned.
+ *              If integer2 is 0, the result has no decimal point or fractional part.
+ *              integer2 can be negative, making the integer2 digit to the left of the decimal point of the value zero.
+ *              This function can also be used by passing only one numeric1 argument without setting Integer2.
+ *              If Integer2 is not set, Integer2 is 0
+ */
+@TransformFunction(names = {"truncate"})
+public class TruncateFunction implements ValueParser {
+
+    private ValueParser bigDecimalParser;
+
+    private ValueParser integerParser;
+
+    public TruncateFunction(Function expr) {
+        List<Expression> expressions = expr.getParameters().getExpressions();
+        if (expressions != null) {
+            bigDecimalParser = OperatorTools.buildParser(expressions.get(0));
+            if (expressions.size() == 2) {
```

Comment:
It is acceptable for "expressions.size() > 2"

Reply:
done